in_text/text_data_release.yml

```diff
@@ -3,14 +3,28 @@ title: >-
 
 
 abstract: >-
-  This dataset provides estimates water temperature from the process-based model, 
-  General Lake Model verion 3 (Hipsey et al. 2019) using calibrated model 
-  configurations and uncalibrated model configurations for 8 reservoirs in 
-  Missouri, USA.  This dataset includes spatial data, observation data, and driver
-  data to run the models.
+  Lake temperature is an important environmental metric for understanding 
+  habitat suitability for many freshwater species and is especially useful
+  when temperatures are predicted throughout the water column (known as
+  temperature profiles). This dataset provides estimates of water temperature
+  at half meter depths for eight reservoirs in Missouri, USA using version 3 
+  of the General Lake Model (Hipsey et al. 2019). The reservoirs are [INSERT NAMES]
+  
+  Both calibrated and uncalibrated model configurations (see '*_glm_nml.zip' 
+  files), as well as, predicted temperatures (see '*_profile_results.zip' files) 
+  for each of the eight reservoirs are included. In addition, the spatial, 
+  observational, and climatic driver datasets ('study_lakes.zip', 'temperature_obs.zip', 
+  and 'NLDAS_GLM_csv.zip', respectively) required to run the models are available. 
+  Lastly, a crosswalk between the reservoir names, the National Hydrography 
+  Dataset High Resolution (NHDHR) identification codes, and identifiers defined by
+  the University of Missouri is provided (see the 'mo_nhdhr_xwalk.csv' file).
+  
+  This research was funded by the Department of the Interior Northeast and Midwest 
+  Climate Adaptation Science Centers with addition financial support from the
+  Midwest Glacial Lakes Partnership and the Minnesota Department of Natural Resources.
```

Comment:
I'd love Jordan's eyes on this funding statement. If changed, we need to also change the duplicate statement near the bottom of this `.yml` file. Maybe ping him via MS Teams and just give the context that it is the 8-lake data release. I created this after reading the two proposals we have in SharePoint, but I think he should verify the statement.

Reply:
@jordansread please check!

Reply:
The funding for this one is solely from the MW CASC project, so the other sources of funding should be removed.

Reply:
Added as an issue

Reply:
Added as an issue